src/map/utils/battleutils.cpp

```diff
@@ -4040,7 +4040,7 @@ namespace battleutils
         {
             case TYPE_PC:
             {
-                if (PDefender->animation == ANIMATION_SIT)
+                if (PDefender->animation == ANIMATION_SIT || (PDefender->animation >= ANIMATION_SITCHAIR_0 && PDefender->animation <= ANIMATION_SITCHAIR_10))
```

Comment:
wait, this check was in `TakeSkillchainDamage`? I am struggling to comprehend why it was here to begin with, but the change is fine